project_stage_state/project.py

```diff
@@ -36,4 +36,5 @@ class ProjectTaskType(models.Model):
 
 class ProjectTask(models.Model):
     _inherit = 'project.task'
-    state = fields.Selection(related='stage_id.state', store=True)
+    state = fields.Selection(
+        related='stage_id.state', store=True, readonly=True)
```

Comment:
It's in the [commit](https://github.com/dreispt/project-service/commit/3bb72abb62fbe9757b528e5d556cb2f7d5dc2fcb) message